src/session.rs

```diff
@@ -105,9 +100,11 @@ pub fn start(config: &Config, initial_request: Option<String>) -> i32 {
                 match controllers.entry(route.clone()) {
                     Entry::Occupied(controller_entry) => {
                         if let Err(err) = controller_entry.get().worker.sender().send(request.clone())  {
-                            if let Some(fifo) = request.meta.fifo {
-                                cancel_blocking_request(fifo);
-                            }
+                            return_request_error(
+                                editor.to_editor.sender(),
+                                &request,
+                                "Language server is no longer running"
```

Comment:
Right, this is definitely an error scenario where it seems good to show the error (I believe this won't happen on a mere timeout).